ci/jenkins/Jenkinsfile_py3-master_gpu_doc

```diff
@@ -74,7 +74,7 @@ core_logic: {
                   --name GluonNLP-${env.BRANCH_NAME}-${env.BUILD_NUMBER} \
                   --save-path batch/${env.BRANCH_NAME}/${env.BUILD_NUMBER}/docs/examples \
                   --work-dir . --source-ref refs/pull/${env.CHANGE_ID}/head \
-                  --command \"(python3 docs/md2ipynb.py ${md_file} | tee ${stdout_file}) 3>&1 1>&2 2>&3 | tee ${stderr_file} \"
+                  --command \"(python3 docs/md2ipynb.py ${md_file} True | tee ${stdout_file}) 3>&1 1>&2 2>&3 | tee ${stderr_file} \"
```

Comment:
I have modified this file now. But CI is still complaining. Is there still anything hidden or anything different from normal python arguments? @leezu 
Thanks!

Reply:
This should be the correct place to modify. I suggest to add some print statements in md2ipynb.py to check how `sys.argv` looks like based on this command

Reply:
I have restarted the CI. Would you please have a look at it?
I have tried to print in the python file (see [this commit](https://github.com/dmlc/gluon-nlp/pull/1179/commits/d6eb142a3e4860f0b651c451a1bb46ef6329b51b)), but it seems that the CI console cannot display it?

Reply:
The CI does display it:

```
[2020-03-21T18:11:50.571Z] + cat docs/examples/language_model/train_language_model.stdout.log

[2020-03-21T18:11:50.571Z] ['docs/md2ipynb.py', 'docs/examples/language_model/train_language_model.md']
```

Reply:
According to the AWS Console, `./gluon_nlp_job.sh refs/pull/1179/head docker/py3 . (python3 docs/md2ipynb.py docs/examples/language_model/train_language_model.md | tee docs/examples/language_model/train_language_model.stdout.log) 3>&1 1>&2 2>&3 | tee docs/examples/language_model/train_language_model.stderr.log ./docs/examples batch/PR-1179/14/docs/examples https://github.com/dmlc/gluon-nlp` is invoked. So for some reason the CI is using the old `ci/jenkins/Jenkinsfile_py3-master_gpu_doc` and not the `ci/jenkins/Jenkinsfile_py3-master_gpu_doc` from your PR.

Reply:
That's consistent with the CI log: https://ci.mxnet.io/blue/organizations/jenkins/GluonNLP-py3-master-gpu-doc/detail/PR-1179/14/pipeline#step-157-log-77

"True" does not show up in the CI log.

Reply:
Cause

> This is due to the fact the author of the Pull Request is not trusted. In that case Jenkins falls back to the Jenkinsfile of the base branch.
> 
> Changing the Jenkinsfile is equivalent to changing the job configuration. Only users with a certain level of trust should be allowed to do it.
> 
> Jenkins handles it that way: when changes to the Jenkinsfile are committed to a PR , Jenkins asks GitHub whether the PR author has the permission to push to the origin repository, if yes they are trusted, otherwise they are not.
> 
> Therefore there are 2 ways to handle this:
> 
>     Give push access (i.e. Write permission) to the PR author on the repository in GitHub
>     Explain to users that changing the Jenkinsfile requires a certain level of permission in GitHub

https://support.cloudbees.com/hc/en-us/articles/227246367-Why-Jenkinsfile-changes-Are-Not-Reflected-in-PR-Build

Reply:
I marked you as authorized user. Let's try again.

Reply:
Wow, that's really a complicated cause and I have never heard of it before. Thanks a lot!

Reply:
Can you merge the master branch to include this fix? https://github.com/dmlc/gluon-nlp/commit/d9641482ec6b72e49733ed8fbce0bfafe2da1562

Reply:
Hello Leonard, I have merged the remote master to include this fix. But it seems that it is still not working in the latest build? Maybe you could lend a hand about it?
Thanks a lot!

Reply:
Seems Jenkins doesn't realize that you are an authorized user now. Can you close this PR and submit a new one?